c2rust-analyze/src/main.rs

```diff
@@ -493,6 +497,30 @@ fn run(tcx: TyCtxt) {
         info.lasn.set(lasn);
     }
 
+    // For testing, putting #[c2rust_analyze_test::fixed_signature] on a function makes all
+    // pointers in its signature FIXED.
+    for &ldid in &all_fn_ldids {
+        if !has_test_attr(tcx, ldid, "fixed_signature") {
```

Comment:
Could we make an `enum` for all the possible attributes?

Reply:
That way we avoid any little typos and can put the docs for each attribute there.

Reply:
I went ahead and did this, even though I don't think it's really necessary.  Typos here seem likely to be caught quickly (if you're adjusting the test attributes, it's probably because you're working on a test and will be checking its behavior closely), and `git grep foo` already got you pretty close to the comment describing the `foo` attribute.